assets/css/main.css

```diff
@@ -12,8 +12,19 @@ body {
     font-family: "Inter", sans-serif;
 }
 
+#navbar-brand-logo {
+    height: 2em;
+    padding: 0.5em;
+}
+
 #main-section {
     background: rgba(17, 15, 15, 1);
+    height: calc(100vh - 3.5em);
```

Comment:
I think this should be updated to be
```CSS
min-height: calc(100vh - 3.5em);
```
This will allow resizing for lengthier pages, while ensuring that shorter pages also span the entire viewport in terms of height.

Reply:
This is resolved in commit https://github.com/umass-cicsoft/umass-cicsoft.github.io/pull/19/commits/b1586eb02106ee9912759baeb17873feaf2dd08c